### DISCUSSION TITLE
求助有没有好用的免费梯子推荐

### DISCUSSION THREAD

**Discussion by Jiesaepe** [share121/inter-knot#267] [其他]
![Uploading Screenshot_20240801_040210.jpg…]()
抛瓦镇楼
如题，最近题主的梯子不能用了，求个推荐，要是能发链接就最好了

**Comment by Zhishirur** [comment]
Clash meta内核订阅链接 普通clash不行
https://igdux.top/bnZf

**Reply by Jiesaepe** [reply]
不行啊，点进去就是404啊

**Comment by sekiroissekiko** [comment]
有一个，但是更新需要在谷歌商店上，你可以去应用网站搜下vpnify先安上过渡一下

**Reply by sekiroissekiko** [reply]
其实不更也行，能下下来就可以
